packages/wonder-blocks-modal/components/modal-title-bar.js

```diff
@@ -25,83 +18,17 @@ type Props = {
     subtitle?: string,
 
     color: "light" | "dark",
-    style?: any,
 };
 
 export default class ModalTitleBar extends React.Component<Props> {
     static defaultProps = {
         color: "light",
     };
 
-    _renderTitleAndSubtitle() {
-        const {title, subtitle} = this.props;
-
-        if (subtitle) {
-            return (
-                <View>
-                    <HeadingSmall id="wb-modal-title">{title}</HeadingSmall>
-                    <LabelSmall>{subtitle}</LabelSmall>
-                </View>
-            );
-        } else {
-            return <HeadingMedium id="wb-modal-title">{title}</HeadingMedium>;
-        }
-    }
-
     render() {
-        const {color, style} = this.props;
-        return (
-            <View
-                style={[
-                    styles.titlebar,
-                    color === "dark" && styles.dark,
-                    (mediaSize) => mediaSize === "small" && styles.small,
-                    style,
-                ]}
-            >
-                <View style={styles.titleAndSubtitle}>
-                    {this._renderTitleAndSubtitle()}
-                </View>
-            </View>
-        );
+        const {color, title, subtitle} = this.props;
+
+        // TODO(scottgrant): Set size appropriately using mediaSize.
+        return <Toolbar color={color} title={title} subtitle={subtitle} />;
```

Comment:
I wonder if we even need the `ModalTitleBar` component now.  We could just use `Toolbar` directly.